packages/plugins/typescript/type-graphql/src/index.ts

```diff
@@ -19,9 +19,9 @@ export const plugin: PluginFunction<TypeGraphQLPluginConfig> = (schema: GraphQLS
   const introspectionDefinitions = includeIntrospectionDefinitions(schema, documents, config);
   const scalars = visitor.scalarsDefinition;
 
-  let definitions = visitorResult.definitions;
+  const definitions = visitorResult.definitions;
   // Sort output by interfaces first, classes last to prevent TypeScript errors
-  definitions.sort((definition1, definition2) => (isDefinitionInterface(definition1) ? -1 : 1));
+  definitions.sort((definition1, definition2) => +isDefinitionInterface(definition2) - +isDefinitionInterface(definition1));
```

Comment:
Compare function should return the same value for the same elements. If compare result for `(definition1, definition2)` is `1`, then result for `(definition2, definition1)` should be `-1`